Sources/XcodeGenKit/GraphVizGenerator.swift

```diff
@@ -0,0 +1,53 @@
+import DOT
+import Foundation
+import GraphViz
+import ProjectSpec
+
+extension Dependency {
+    var graphVizName: String {
+        switch self.type {
+        case .bundle, .package, .sdk, .framework, .carthage:
+            return "<<external>>\\n\(reference)"
```

Comment:
How about including the name of the dependency type instead of external? Same as enum cases but strings
```
[carthage]
Alamofire
```

Reply:
Updated [here](https://github.com/yonaskolb/XcodeGen/pull/852/commits/e32078dae8486069d4f3c30f0f932da52dc5e30e)